src/workerd/jsg/modules.h

```diff
@@ -177,13 +177,11 @@ class ModuleRegistry {
 
     enum class CompileFlags {
```

Comment:
If this is still intended to be "flags", then I still would like the values to be declared explicitly as bits. If this is no longer expected to behave as a flag set then I think we should rename the type.